src/bokeh/core/property/bases.py

```diff
@@ -247,19 +248,19 @@ def matches(self, new: T, old: T) -> bool:
 
         """
         import numpy as np
-        import pandas as pd
 
         if isinstance(new, np.ndarray) or isinstance(old, np.ndarray):
             return np.array_equal(new, old)
 
-        if isinstance(new, pd.Series) or isinstance(old, pd.Series):
-            return np.array_equal(new, old)
+        if uses_pandas(new) or uses_pandas(new):
```

Comment:
Was this supposed to be
```suggestion
        if uses_pandas(new) or uses_pandas(old):
```
?